Pipfile

```
@@ -19,7 +19,7 @@ wrapt = "~=1.15"
 [packages]
 policyuniverse = "==1.5.1.20230817"
 requests = "==2.31.0"
-panther-analysis-tool = "~=0.36"
+panther-analysis-tool = "~=0.38"
 panther-detection-helpers = ">=0.1.2"
```

I'm working on an RFC to simplify this process, for now let's also pin the new PDH version 🙏 

```suggestion
panther-detection-helpers = "==0.2.0"
```

---

Thanks! Addressed in [`a90cf5f` (#1079)](https://github.com/panther-labs/panther-analysis/pull/1079/commits/a90cf5f2b70b13e300921984f65406645b186c2a).